scripts/bug_retriever.py

```diff
@@ -60,7 +60,8 @@ def retrieve_bugs(self):
 
         # Try to re-download inconsistent bugs, up to three times.
         for i in range(3):
-            bug_ids = set(bug_snapshot.get_inconsistencies())
+            bugs = bugzilla.get_bugs()
+            bug_ids = set(bug_snapshot.get_inconsistencies(bugs))
```

Comment:
This is still downloading all bugs

Reply:
Can you please help me with some extra details ?

Reply:
I've explained what needs to be done in https://github.com/mozilla/bugbug/issues/593#issue-456315165, feel free to ask in that issue about any doubts you might have.